lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -0,0 +1,938 @@
+
+  app->start;
+
+L<Mojolicious::Plugin::Config> is a built-in plugin which can populate L<Mojolicious/"config"> using a config file.
+Plugins can also set up routes, hooks, handlers, or even load other plugins. A list of built-in plugins can be found at
+L<Mojolicious::Plugins/"PLUGINS">, and many more are available from
+L<CPAN|https://metacpan.org/search?q=Mojolicious+Plugin>.
+
+=head2 Placeholders
+
+Route placeholders allow capturing parts of a request path until a C</> or C<.> separator occurs, similar to the
+regular expression C<([^/.]+)>. Results are accessible via L<Mojolicious::Controller/"stash"> and
+L<Mojolicious::Controller/"param">.
+
+  use Mojolicious::Lite -signatures;
+
+  # /foo/test
+  # /foo/test123
+  get '/foo/:bar' => sub ($c) {
+    my $bar = $c->stash('bar');
+    $c->render(text => "Our :bar placeholder matched $bar");
+  };
+
+  # /testsomething/foo
+  # /test123something/foo
+  get '/<:bar>something/foo' => sub ($c) {
+    my $bar = $c->param('bar');
+    $c->render(text => "Our :bar placeholder matched $bar");
+  };
+
+  app->start;
+
+To separate them from the surrounding text, you can surround your placeholders with C<E<lt>> and C<E<gt>>, which also
+makes the colon prefix optional.
+
+=head2 Relaxed Placeholders
+
+Relaxed placeholders allow matching of everything until a C</> occurs, similar to the regular expression C<([^/]+)>.
+
+  use Mojolicious::Lite;
+
+  # /hello/test
+  # /hello/test.html
+  get '/hello/#you' => 'groovy';
+
+  app->start;
+  __DATA__
+
+  @@ groovy.html.ep
+  Your name is <%= $you %>.
+
+=head2 Wildcard placeholders
+
+Wildcard placeholders allow matching absolutely everything, including C</> and C<.>, similar to the regular expression
+C<(.+)>.
+
+  use Mojolicious::Lite;
+
+  # /hello/test
+  # /hello/test123
+  # /hello/test.123/test/123
+  get '/hello/*you' => 'groovy';
+
+  app->start;
+  __DATA__
+
+  @@ groovy.html.ep
+  Your name is <%= $you %>.
+
+=head2 HTTP methods
+
+Routes can be restricted to specific request methods with different keywords like L<Mojolicious::Lite/"get"> and
+L<Mojolicious::Lite/"any">.
+
+  use Mojolicious::Lite -signatures;
+
+  # GET /hello
+  get '/hello' => sub ($c) {
+    $c->render(text => 'Hello World!');
+  };
+
+  # PUT /hello
+  put '/hello' => sub ($c) {
+    my $size = length $c->req->body;
+    $c->render(text => "You uploaded $size bytes to /hello.");
+  };
+
+  # GET|POST|PATCH /bye
+  any ['GET', 'POST', 'PATCH'] => '/bye' => sub ($c) {
+    $c->render(text => 'Bye World!');
+  };
+
+  # * /whatever
+  any '/whatever' => sub ($c) {
+    my $method = $c->req->method;
+    $c->render(text => "You called /whatever with $method.");
+  };
+
+  app->start;
+
+=head2 Optional placeholders
+
+All placeholders require a value, but by assigning them default values you can make capturing optional.
+
+  use Mojolicious::Lite -signatures;
+
+  # /hello
+  # /hello/Sara
+  get '/hello/:name' => {name => 'Sebastian', day => 'Monday'} => sub ($c) {
```

Comment:
Oh, am I using `app->default` here and not even realize it??  Would be great to link the word "default" on L430 above to `Mojolicious/"default"`.